packages/venia-ui/lib/components/ShoppingBag/shoppingBag.js

```diff
@@ -21,23 +29,41 @@ const ShoppingBag = props => {
     useScrollLock(isOpen);
 
     const talonProps = useShoppingBag({
-        setIsOpen
+        setIsOpen,
+        ...ShoppingBadOperations
     });
 
-    const { onDismiss } = talonProps;
+    const {
+        onDismiss,
+        productListings,
+        loading,
+        error,
+        totalQuantity,
+        handleRemoveItem
+    } = talonProps;
 
     const classes = mergeClasses(defaultClasses, props.classes);
     const rootClass = isOpen ? classes.root_open : classes.root;
 
+    if (error) {
+        return <Error error={error} />;
+    }
+
     return (
         <aside className={rootClass}>
             {/* The Mask. */}
             <button className={classes.mask} onClick={onDismiss} type="reset" />
             {/* The Contents. */}
             <div className={classes.contents}>
-                <div className={classes.header}>Header TBD</div>
+                <div
+                    className={classes.header}
+                >{`Header TBD total quantity: ${totalQuantity}`}</div>
```

Comment:
I have added `totalQuantity` to the GQL query as well. It is not needed in this PR but helpful for the tickets in the shopping bag epic.